src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length)];
```

Comment:
Please make such complicated constructions easier to read:
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index];
```